datafusion/core/src/execution/context.rs

```diff
@@ -1764,7 +1784,7 @@ impl ContextProvider for SessionState {
             Ok(schema) => {
                 let provider = schema.table(resolved_ref.table).ok_or_else(|| {
                     DataFusionError::Plan(format!(
-                        "'{}.{}.{}' not found",
+                        "table '{}.{}.{}' not found",
```

Comment:
`not found` is a hard error message to ctrl-f for. Adding the word `table` will hopefully make this statistically more likely to be found.